contrib/packs/actions/install.meta.yaml

```diff
@@ -37,3 +37,9 @@
       description: "Set to True to skip pack dependency installations."
       required: false
       default: false
+    timeout:
+      default: 600
```

Comment:
What was the default timeout before this PR?

Reply:
the default was 600 only.

Reply:
So the default was 600, now it's 600? No change?

Reply:
Users should have been able to override this parameter before. I don't see the what we gain by making it explicit if we're just going to keep the same default value. @hnanchahal ?